### DISCUSSION TITLE
Player issue with extra MPlayer/MPV parameters in Windows version

### DISCUSSION THREAD

**Discussion by custardo1** [smplayer-dev/smplayer#587] [Q&A] [answered]
I try to enter some parameters in Advanced->MPlayer/MPV->Options, but never mind what I enter, after that the video will not start (as in it looks like the player instance isn't created). Even the parameters supplied in the Help (-flip -nosound) don't work. Any ideas? This is at least the case with version 22.2.0 and 22.7.0. 64bit

**Comment by smplayer-dev** [comment, marked as answer]
SMPlayer uses by default mpv. If you use options for mplayer it won't work. The options for mpv: https://mpv.io/manual/stable/#options

**Reply by custardo1** [reply]
Thanks, that did the trick.
